src/unix/udp.c

```diff
@@ -1021,6 +1134,11 @@ int uv_udp_getsockname(const uv_udp_t* handle,
                        struct sockaddr* name,
                        int* namelen) {
 
+#if defined(DISCORD_ENABLE_NETMAP)
```

Comment:
Do you need to do this stuff inside all the other `uv__udp_` methods? Namely the `_multicast_` flavors? Perhaps just `assert(!handle->use_netmap);`?

Reply:
my intention was that those will eventually work into `uv__udp_netmap` methods by way of `uv__setsockopt`